src/SpotifyApi.NetCore/LibraryApi.cs

```diff
@@ -0,0 +1,410 @@
+        /// <returns>A Task that, once successfully completed, returns a full <see cref="PagedShows"/> object.</returns>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/library/get-users-saved-shows/
+        /// </remarks>
+        public async Task<PagedShows> GetUserSavedShows(
+            int limit = 20,
+            int offset = 0,
+            string accessToken = null
+            ) => await GetUserSavedShows<PagedShows>(limit, offset, accessToken);
+
+        /// <summary>
+        /// Get User's Saved Shows
+        /// </summary>
+        /// <param name="limit">Optional. The maximum number of objects to return. Default: 20. Minimum: 1. Maximum: 50.</param>
+        /// <param name="offset">Optional. The index of the first object to return. Default: 0 (i.e., the first object). Use with limit to get the next set of objects.</param>
+        /// <returns>A Task that, once successfully completed, returns a full <see cref="PagedShows"/> object.</returns>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/library/get-users-saved-shows/
+        /// </remarks>
+        public async Task<T> GetUserSavedShows<T>(
+            int limit = 20,
+            int offset = 0,
+            string accessToken = null
+            )
+        {
+            if (limit < 1 || limit > 50) throw new
+                    ArgumentException("The limit can be a minimum of 1 and a maximum of 50.");
+            if (offset < 0) throw new
+                     ArgumentException("The offset must be an integer value greater than 0.");
+
+            var builder = new UriBuilder($"{BaseUrl}/me/albums");
+            builder.AppendToQuery("limit", limit);
+            builder.AppendToQuery("offset", offset);
+            return await GetModel<T>(builder.Uri, accessToken);
+        }
+        #endregion
+
+        #region GetUserSavedTracks
+        /// <summary>
+        /// Get a list of the songs saved in the current Spotify user’s ‘Your Music’ library.
+        /// </summary>
+        /// <param name="limit">Optional. The maximum number of objects to return. Default: 20. Minimum: 1. Maximum: 50.</param>
+        /// <param name="offset">Optional. The index of the first object to return. Default: 0 (i.e., the first object). Use with limit to get the next set of objects.</param>
+        /// <param name="market">Optional. An ISO 3166-1 alpha-2 country code or the string from_token. Provide this parameter if you want to apply Track Relinking.</param>
+        /// <returns>A Task that, once successfully completed, returns a full <see cref="PagedTracks"/> object.</returns>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/library/get-users-saved-tracks/
+        /// </remarks>
+        public async Task<PagedTracks> GetUserSavedTracks(
```

Comment:
* Method name = `GetUsersSavedTracks`
* Remove `async` and `await` keywords here